src/qml/CustomControls/Slider.qml

```diff
@@ -14,6 +14,7 @@ CustomTouchArea {
 	property real indicator: 0.0
     property bool useIndicator: false
     property bool midiMappingEnabled: true
+    property bool feedbackEnabled: true
```

Comment:
Where are you using this property? Maybe I overlooked it...

Reply:
I added an option to FaderBankBlock to disable midi feedback. The rationale for me was that with multiple midi devices connected (korg nano + behringer x-touch mini) the feedback includes strange lagging in midi processing. Disabling feedback globally is an option but you do not have button leds then...